### ISSUE TITLE
Unexpected method 'appcast' called on Cask near-lock.

### ISSUE THREAD

**Issue by SkyLee365** [Homebrew/homebrew-cask#173357]
### Verification

- [X] I understand that [if I ignore these instructions, my issue may be closed without review](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/closing_issues_without_review.md).
- [X] I have retried my command with `--force`.
- [X] I ran `brew update-reset && brew update` and retried my command.
- [X] I ran `brew doctor`, fixed as many issues as possible and retried my command.
- [X] I have checked the instructions for [reporting bugs](https://github.com/Homebrew/homebrew-cask#reporting-bugs).
- [X] I made doubly sure this is not a [checksum does not match / SHA256 mismatch](https://docs.brew.sh/Common-Issues#cask---checksum-does-not-match) error (do not open an issue before trying to open a PR to fix first).

### Description of issue

Error: Unexpected method 'appcast' called on Cask near-lock.
Follow the instructions here:
  https://github.com/Homebrew/homebrew-cask#reporting-bugs
Error: Unexpected method 'appcast' called on Cask near-lock.
Follow the instructions here:
  https://github.com/Homebrew/homebrew-cask#reporting-bugs

### Command that failed

brew outdated

### Output of command with `--verbose --debug`

```shell
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromInstalledPathLoader): loading near-lock
Error: Unexpected method 'appcast' called on Cask near-lock.
Follow the instructions here:
  https://github.com/Homebrew/homebrew-cask#reporting-bugs
```

### Output of `brew doctor` and `brew config`

```shell
brew doctor
Error: Unexpected method 'appcast' called on Cask near-lock.
Follow the instructions here:
  https://github.com/Homebrew/homebrew-cask#reporting-bugs
Error: Unexpected method 'appcast' called on Cask near-lock.
Follow the instructions here:
  https://github.com/Homebrew/homebrew-cask#reporting-bugs
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  openssl@1.1

brew config
HOMEBREW_VERSION: 4.2.21-71-g3a36df5
ORIGIN: https://github.com/Homebrew/brew
HEAD: 3a36df5f2e041dc4d89c3b7afd290602c5d2876c
Last commit: 2 hours ago
Core tap HEAD: 4f3d0836ab01a4f994331793830b4b166adf3206
Core tap last commit: 18 minutes ago
Core tap JSON: 09 May 15:45 UTC
Core cask tap HEAD: b1cd431d57e5cd1aebe4db5e8782f62721e05a50
Core cask tap last commit: 8 minutes ago
Core cask tap JSON: 09 May 15:45 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 4
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 3.1.4 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: quad-core 64-bit broadwell
Clang: 14.0.0 build 1400
Git: 2.45.0 => /usr/local/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 12.7.4-x86_64
CLT: 14.2.0.0.1.1668646533
Xcode: N/A
```

### Output of `brew tap`

```shell
brew tap                                  
buo/cask-upgrade
gcenx/wine
homebrew/cask
homebrew/core
homebrew/services
```

**Comment by SMillerDev** [comment]
Duplicate of https://github.com/Homebrew/homebrew-cask/issues/173201
